packages/okta-react-native/android/src/main/java/com/oktareactnative/OktaSdkBridgeModule.java

```diff
@@ -347,6 +347,22 @@ public void onError(String e, AuthorizationException error) {
         }
     }
 
+    @ReactMethod
+    public void clearTokens(final Promise promise) {
+        try {
+            if (webClient == null) {
+                promise.reject(OktaSdkError.NOT_CONFIGURED.getErrorCode(), OktaSdkError.NOT_CONFIGURED.getErrorMessage());
+                return;
+            }
+
+            final SessionClient sessionClient = webClient.getSessionClient();
+            sessionClient.clear();
+            promise.resolve(true);
```

Comment:
This will clear the `webClient`. The `authClient` needs to be checked and cleared too.
The difference between the two clients is webclient uses the SIW and authClient uses a custom UI.

Reply:
@FeiChen-okta updated to clear both `webClient` and `authClient`. I tested functionality using `samples-js-react-native` for both custom & browser sign in. After clearing tokens, operations such as `refreshTokens` or `getAccessToken` will fail.